.openshift-ci/google-cloud-sdk/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+GOOGLE_CLOUD_SDK_VERSION=383.0.0
+
+install () {
+  # Set sudo to work whether logged in as root user or non-root user
+  if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; w
```

Comment:
I'm failing to see why this line is needed, would you mind elaborating a little? 🙂

Reply:
It's mostly because I have no idea in which environment is it going to be running (w/ or w/o root), so I reckon it's not a bad idea. Any downsides?

Reply:
None that I can see, it just called my attention that `SUDO` is not being used anywhere after being exported.